Octokit/Clients/RepositoriesClient.cs

```diff
@@ -86,6 +86,11 @@ public Task<Repository> Create(string organizationLogin, NewRepository newReposi
                     errorMessage,
                     StringComparison.OrdinalIgnoreCase))
                 {
+                    if (string.IsNullOrEmpty(organizationLogin))
+                    {
+                        throw new RepositoryExistsException(newRepository.Name, e);
+                    }
```

Comment:
In looking at this, shouldn't we always be passing in `baseAddress` to `RepositoryExistsException`. Why would we only pass it in if we know your org?